docs/_static/gammapy.css

```diff
@@ -1,11 +1,180 @@
+/*
+* Most code incorporated from bootstrap-astropy.css file in Astropy Project
```

Comment:
You are adding 185 lines of CSS in `docs/_static/gammapy.css`.
That's OK, but if it's possible to have less CSS with the same result (e.g. by inheriting CSS from the RTD theme), please make that change.
In Gammapy we're good at Python, but not CSS, so we should aim to have as little CSS as possible to be easy to maintain for the coming years.
Also a short mention in the dev docs how the docs theme works might be useful. E.g. it's not clear to me why you mention `bootstrap-astropy.css` here - do we still include their CSS or is this a modified copy of a lot of their CSS? Given that we use the RTD theme now, it's not obvious to me why we still need the astropy CSS!?